drivers/bridge/bridge.go

```diff
@@ -92,9 +92,14 @@ func init() {
 	portMapper = portmapper.New()
 }
 
-// New provides a new instance of bridge driver
-func New(dc driverapi.DriverCallback) (string, driverapi.Driver) {
-	return networkType, &driver{}
+// New constructs a new bridge driver
+func New() driverapi.Driver {
+	return &driver{}
+}
+
+// Init registers a new instance of bridge driver
+func Init(dc driverapi.DriverCallback) error {
```

Comment:
The name is a matter of taste I think -- I prefer `Init` to `New` because it's now invoked for the side-effect, rather than returning a value.

It's not necessary for `bridge.New` to be exported though, quite right. It's just to avoid directly constructing driver structs in the tests. I'll rename it to `newDriver`.